Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -172,30 +167,30 @@ extension WebSocket {
         }
     }
     
-    func disconnect() {
+    func disconnect(reason: String) {
```

Comment:
I'd say a `WebSocketDisconnectReason` enum is needed here

Reply:
I didn't think about that when I worked on a fix. I had to find a way to see the flow with disconnection from different points and they all different, so it won't be needed to reuse (like you suggest use enum). I like the current solution with the freedom to set a const string directly in the interesting points.